### PR TITLE
will you look over my tests?

### PR DESCRIPTION
**Pull request by katyjane8**
I want to make sure I include edge cases. Thank you!!

### CODE REVIEW

File: test/event_reporter_test.rb [hunk 1]
```diff
@@ -16,35 +16,28 @@ def test_reporter_can_find_all_Johns
...
-    refute er.list.empty?
-
-    list.clear_queue
-    assert_equal 0, list.queue_count
-  end
-
   def test_reporter_can_find_all_Marys_and_queue_them
     er = EventReporter.new('./data/full_event_attendees.csv')
-
     result = er.find_attendees(:first_name, "Mary")
```
Comment (izaxveta) [inline test/event_reporter_test.rb]: you could also assert that the first object in your list ACTUALLY has the first name of Mary.

```assert_equal "Mary", er.list.first.first_name```

File: test/event_reporter_test.rb [hunk 2]
```diff
@@ -16,35 +16,28 @@ def test_reporter_can_find_all_Johns
...
     refute er.list.empty?
   end
 
-  def test_queue_can_be_sorted
+  def test_queue_can_be_sorted_and_first_in_sort_has_20009_zipcode
     er = EventReporter.new('./data/attendees_fixture.csv')
     er.find_attendees(:first_name, "Sarah")
     result = er.sort_queue(:state)
 
     assert_equal "20009", result.first.zipcode
```
Comment (izaxveta) [inline test/event_reporter_test.rb]: if this is sorting the queue by state, you could

```assert_equal "20009", result.first.zipcode```
AND
```assert_equal "<a zipcode greater than 20009>", result.last.zipcode```

to simply show that their zipcode is organized in ascending order.

File: test/event_reporter_test.rb [hunk 3]
```diff
@@ -16,35 +16,28 @@ def test_reporter_can_find_all_Johns
...
   end
 
-  # def test_it_can_print_all_Marys
+  # def test_it_can_print_all_Mary_Kates
+  #   er = EventReporter.new('./data/attendees_fixture.csv')
+  #   er.find_attendees(:first_name, "Mary Kate")
+  #   result = er.print_sorted(:first_name)
+  #
+  #   assert_equal "", result
```
Comment (izaxveta) [inline test/event_reporter_test.rb]: would be better to have more to attendees in your fixture to test from,
but this is a good test to start with.

File: test/event_reporter_test.rb [hunk 4]
```diff
@@ -55,10 +48,10 @@ def test_find_SLC_in_CSV
   end
 
   def test_CSV_can_be_written_with_queue
-    skip
     er = EventReporter.new('./data/full_event_attendees.csv')
+    result = er.find_attendees(:zipcode, "90210")
 
-    assert_equal 45, er.write_list
+    assert_equal 1, er.write_list.count
```
Comment (izaxveta) [inline test/event_reporter_test.rb]: you could also that the one object in that list has the zipcode of "90210" <---ayeeee I see what you did there.

```assert_equal "90210", er.list.first.zipcode```

File: test/queue_test.rb [hunk 5]
```diff
@@ -4,19 +4,28 @@
 
 class ListTest < Minitest::Test
   def test_queue_count_in_event
```
Comment (izaxveta) [inline test/queue_test.rb]: you could be more specific in this test name

```def test_queue_count_starts_at_zero```

Comment (izaxveta) [reply test/queue_test.rb]: make your test names more [imaginary finger quotes] robust [imaginary finger quotes] 😸

File: test/queue_test.rb [hunk 6]
```diff
@@ -4,19 +4,28 @@
...
+    assert_equal 35, er.print_sorted(:last_name).count
+  end
+
+  def test_queue_can_be_cleared
+    er = EventReporter.new('./data/full_event_attendees.csv')
+    list = List.new
+
+    er.find_attendees(:first_name, "Rachel")
+    refute er.list.empty?
+
```
Comment (izaxveta) [inline test/queue_test.rb]: good good 😈

File: test/event_reporter_test.rb (hunk 4 above)
Comment (katyjane8) [reply test/event_reporter_test.rb]: 😄
